src/legacy/amos/dgamln.f

```diff
@@ -168,7 +168,7 @@ DOUBLE PRECISION FUNCTION DGAMLN(Z,IERR)
         S = S + TRM
    30 CONTINUE
    40 CONTINUE
-      IF (ZINC.NE.0.0D0) GO TO 50
+      IF ((ZINC.GT.0.0D0).OR.(ZINC.LT.0.0D0)) GO TO 50
```

Comment:
```suggestion
      IF ((DABS(ZINC) > EPS) GO TO 50
```
And define `EPS` appropriately.

Reply:
Thank you. This is my mistake.

Reply:
I added extra [commit](https://github.com/tkoyama010/fortran-utils/commit/f88f9cf6e9856e397bf181e8dd24d7490a65a6f8).